ratelimits/names.go

```diff
@@ -200,3 +215,9 @@ func nameToEnumString(s Name) string {
 func bucketKey(name Name, id string) string {
 	return nameToEnumString(name) + ":" + id
 }
+
+// isLimitNameValid returns true if the provided Name is a valid rate limit
+// name.
+func isNameValid(name Name) bool {
```

Comment:
`isNameValid` could become a method on `Name` since it's only ever used while working with a `Name`.

Reply:
Great idea.